vyper/opcodes.py

```diff
@@ -1,173 +1,174 @@
-    'POP': [0x50, 1, 0, 2],
-    'MLOAD': [0x51, 1, 1, 3],
-    'MSTORE': [0x52, 2, 0, 3],
-    'MSTORE8': [0x53, 2, 0, 3],
-    'SLOAD': [0x54, 1, 1, 200],
-    'SSTORE': [0x55, 2, 0, 20000],
-    'JUMP': [0x56, 1, 0, 8],
-    'JUMPI': [0x57, 2, 0, 10],
-    'PC': [0x58, 0, 1, 2],
-    'MSIZE': [0x59, 0, 1, 2],
-    'GAS': [0x5a, 0, 1, 2],
-    'JUMPDEST': [0x5b, 0, 0, 1],
-    'PUSH1': [0x60, 0, 1, 3],
-    'PUSH2': [0x61, 0, 1, 3],
-    'PUSH3': [0x62, 0, 1, 3],
-    'PUSH4': [0x63, 0, 1, 3],
-    'PUSH5': [0x64, 0, 1, 3],
-    'PUSH6': [0x65, 0, 1, 3],
-    'PUSH7': [0x66, 0, 1, 3],
-    'PUSH8': [0x67, 0, 1, 3],
-    'PUSH9': [0x68, 0, 1, 3],
-    'PUSH10': [0x69, 0, 1, 3],
-    'PUSH11': [0x6a, 0, 1, 3],
-    'PUSH12': [0x6b, 0, 1, 3],
-    'PUSH13': [0x6c, 0, 1, 3],
-    'PUSH14': [0x6d, 0, 1, 3],
-    'PUSH15': [0x6e, 0, 1, 3],
-    'PUSH16': [0x6f, 0, 1, 3],
-    'PUSH17': [0x70, 0, 1, 3],
-    'PUSH18': [0x71, 0, 1, 3],
-    'PUSH19': [0x72, 0, 1, 3],
-    'PUSH20': [0x73, 0, 1, 3],
-    'PUSH21': [0x74, 0, 1, 3],
-    'PUSH22': [0x75, 0, 1, 3],
-    'PUSH23': [0x76, 0, 1, 3],
-    'PUSH24': [0x77, 0, 1, 3],
-    'PUSH25': [0x78, 0, 1, 3],
-    'PUSH26': [0x79, 0, 1, 3],
-    'PUSH27': [0x7a, 0, 1, 3],
-    'PUSH28': [0x7b, 0, 1, 3],
-    'PUSH29': [0x7c, 0, 1, 3],
-    'PUSH30': [0x7d, 0, 1, 3],
-    'PUSH31': [0x7e, 0, 1, 3],
-    'PUSH32': [0x7f, 0, 1, 3],
-    'DUP1': [0x80, 1, 2, 3],
-    'DUP2': [0x81, 1, 2, 3],
-    'DUP3': [0x82, 1, 2, 3],
-    'DUP4': [0x83, 1, 2, 3],
-    'DUP5': [0x84, 1, 2, 3],
-    'DUP6': [0x85, 1, 2, 3],
-    'DUP7': [0x86, 1, 2, 3],
-    'DUP8': [0x87, 1, 2, 3],
-    'DUP9': [0x88, 1, 2, 3],
-    'DUP10': [0x89, 1, 2, 3],
-    'DUP11': [0x8a, 1, 2, 3],
-    'DUP12': [0x8b, 1, 2, 3],
-    'DUP13': [0x8c, 1, 2, 3],
-    'DUP14': [0x8d, 1, 2, 3],
-    'DUP15': [0x8e, 1, 2, 3],
-    'DUP16': [0x8f, 1, 2, 3],
-    'SWAP1': [0x90, 2, 2, 3],
-    'SWAP2': [0x91, 2, 2, 3],
-    'SWAP3': [0x92, 2, 2, 3],
-    'SWAP4': [0x93, 2, 2, 3],
-    'SWAP5': [0x94, 2, 2, 3],
-    'SWAP6': [0x95, 2, 2, 3],
-    'SWAP7': [0x96, 2, 2, 3],
-    'SWAP8': [0x97, 2, 2, 3],
-    'SWAP9': [0x98, 2, 2, 3],
-    'SWAP10': [0x99, 2, 2, 3],
-    'SWAP11': [0x9a, 2, 2, 3],
-    'SWAP12': [0x9b, 2, 2, 3],
-    'SWAP13': [0x9c, 2, 2, 3],
-    'SWAP14': [0x9d, 2, 2, 3],
-    'SWAP15': [0x9e, 2, 2, 3],
-    'SWAP16': [0x9f, 2, 2, 3],
-    'LOG0': [0xa0, 2, 0, 375],
-    'LOG1': [0xa1, 3, 0, 750],
-    'LOG2': [0xa2, 4, 0, 1125],
-    'LOG3': [0xa3, 5, 0, 1500],
-    'LOG4': [0xa4, 6, 0, 1875],
-    'CREATE': [0xf0, 3, 1, 32000],
-    'CALL': [0xf1, 7, 1, 700],
-    'CALLCODE': [0xf2, 7, 1, 700],
-    'RETURN': [0xf3, 2, 0, 0],
-    'DELEGATECALL': [0xf4, 6, 1, 700],
-    'CALLBLACKBOX': [0xf5, 7, 1, 700],
-    'SELFDESTRUCT': [0xff, 1, 0, 25000],
-    'STATICCALL': [0xfa, 6, 1, 40],
-    'REVERT': [0xfd, 2, 0, 0],
-    'INVALID': [0xfe, 0, 0, 0],
-    'DEBUG': [0xa5, 1, 0, 0]
+opcodes: Dict[str, Tuple[Union[int, None], int, int, int]] = {
```

Comment:
I think the types of `opcodes` and `pseudo_opcodes` can be more specific, and then relax the type when we combine them into `comb_opcodes`.